appcelerator/titanium_mobile
# fix: always load aca module first on startup

- Amend `ti.main.js` to always load Axway Crash Analytics module first on startup

##### TEST CASE
- Make sure Titanium application runs without ACA added.
- Make sure Titanium application runs with ACA added and is able to log crashes.

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-28079)
